packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -221,19 +216,26 @@ if (!process.env.PRIVATE_KEY_USER) {
 const localWallet = new Wallet(process.env.PRIVATE_KEY_CUSTOM)
 const userWallet = new Wallet(process.env.PRIVATE_KEY_USER)
 
-async function deployERC20ToL1() {
-  console.log('Deploying ERC20 to L1...')
-  const contract = new TestERC20__factory().connect(
-    localWallet.connect(parentProvider)
+async function deployERC20ToParentChain() {
+  console.log('Deploying ERC20...')
+  const signer = localWallet.connect(parentProvider)
+  const factory = new ContractFactory(contractAbi, contractByteCode, signer)
+  const l1TokenContract = await factory.deploy(
```

Comment:
Deploy an actual `TESTARB` token on the chain for testing standard-erc20 gateway.